### PR TITLE
Remove addressbook

### PR DESCRIPTION
**Pull request by advaypal**

### CODE REVIEW

**Comment by advaypal** [comment]
Coverage decreased because I removed one test that was checking for invalid addresses.

**Comment by Skaty** [comment]
@advaypal: What test and where?

File: src/test/java/seedu/task/logic/LogicManagerTest.java [hunk 1]
```diff
@@ -142,20 +146,20 @@ public void execute_clear() throws Exception {
...
-    public void execute_add_invalidArgsFormat() throws Exception {
-        String expectedMessage = String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE);
-        assertCommandBehavior(
-                "add Valid Name 12345 e/valid@email.butNoPhonePrefix a/valid, address", expectedMessage);
-        assertCommandBehavior(
-                "add Valid Name p/12345 valid@email.butNoPrefix a/valid, address", expectedMessage);
-        assertCommandBehavior(
-                "add Valid Name p/12345 e/valid@email.butNoAddressPrefix valid, address", expectedMessage);
-    }
+//    @Test
```
Comment (advaypal) [inline src/test/java/seedu/task/logic/LogicManagerTest.java:153]: @Skaty this test

**Review by Skaty — APPROVED** [review]
:+1:. Will merge and allow a coverage drop.

**Pull request merged**